### PR TITLE
Tidy up github files

### PR DESCRIPTION
**Pull request by nabobalis**
Fixes https://github.com/sunpy/sunpy/issues/4522

### CODE REVIEW

File: docs/dev_guide/newcomers.rst [hunk 1]
```diff
@@ -1 +1,269 @@
-.. include:: ../../CONTRIBUTING.rst
+.. _newcomers:
```
Comment (nabobalis) [inline docs/dev_guide/newcomers.rst:1]: Copied this here for now.

Comment (Cadair) [reply docs/dev_guide/newcomers.rst:1]: Isn't this somewhat defeating the purpose? It's fine for now though.

Comment (nabobalis) [reply docs/dev_guide/newcomers.rst:1]: Sure it is but I don't have a better idea.

Comment (Cadair) [reply docs/dev_guide/newcomers.rst:1]: yeah I guess the only real alternative is to use MyST and download it on build.

Comment (nabobalis) [reply docs/dev_guide/newcomers.rst:1]: I will do that.

Comment (Cadair) [reply docs/dev_guide/newcomers.rst:1]: Which I am not sure I am on board with. That being said maybe we should re-write `CONTRIBUTING.md` to be more generic and higher level and link to this?

Comment (Cadair) [reply docs/dev_guide/newcomers.rst:1]: yeah maybe, I wonder if anyone else has opinions.

Comment (nabobalis) [reply docs/dev_guide/newcomers.rst:1]: Will make the project wide one the same as the website.

**Review by Cadair — APPROVED** [review]

**Pull request merged**